core/src/test/java/org/apache/iceberg/TestFormatVersions.java

```diff
@@ -28,51 +28,79 @@
 public class TestFormatVersions extends TestBase {
   @Parameters(name = "formatVersion = {0}")
   protected static List<Object> parameters() {
-    return Arrays.asList(1);
+    return Arrays.asList(1, 2);
   }
 
   @TestTemplate
   public void testDefaultFormatVersion() {
```

Comment:
This seems wrong to me? Or at least it looks wrong, why would testDefaultFormatVersion by between 1 and 2?

Reply:
I think this test is not named well. `table.ops().current().formatVersion()` is parameterized on the list of versions from above. I don't think this test is adding anything so we can just remove it

Reply:
Removed the test

Reply:
the test should be added back and updated to 
```
@TestTemplate
  public void testDefaultFormatVersion() {
    assertThat(table.ops().current().formatVersion()).isEqualTo(formatVersion);
  }
```

Reply:
Added back

Reply:
Yeah it's really testing the formatVersion API which we should keep to make sure behavior stays the same no matter how many version there are!